src/viv-imx6/view-backend.cpp

```diff
@@ -79,7 +78,18 @@ ViewBackend::~ViewBackend()
 
 void ViewBackend::initialize()
 {
-    wpe_view_backend_dispatch_set_size(backend, width, height);
+    uint32_t w = WIDTH, h = HEIGHT;
+    char *tmp;
+
+    if (tmp = std::getenv("WPE_INIT_VIEW_WIDTH")) {
+        w = atoi(tmp);
+        this->width = { w }; };
```

Comment:
This is some funky indentation, the last closing brace should be at the same level than the if above